.github/workflows/build.yml

```diff
@@ -8,6 +8,7 @@ on:
 
 env:
   CARGO_TERM_COLOR: always
+  DATABASE_URL: "sqlite::memory:"
```

Comment:
I think we're going to need the stuff from the offline prepare?

Reply:
Yes!